jsonschema-core.md

```diff
@@ -523,11 +523,29 @@ to inform the user of the evaluation path that produced the error or annotation.
 
 ### Keyword Interactions
 
-Keyword behavior MAY be defined in terms of the annotation results of
-[subschemas](#root) and/or adjacent keywords (keywords within the same schema
-object) and their subschemas. Such keywords MUST NOT result in a circular
-dependency. Keywords MAY modify their behavior based on the presence or absence
-of another keyword in the same [schema object](#schema-document).
+Unless otherwise specified, keywords act independently.
+
+Keywords MAY modify their behavior based on the presence, absence, or value of
+another keyword in the same schema object. Such keywords MUST NOT result in a
+circular dependency.
+
+Supplementary specifications are encouraged to specify any dependencies as part
+of the dependent keyword (i.e. the keyword whose behavior is modified).
+
+Within this document, keyword dependencies are expressed using one of the
+following mechanics:
```

Comment:
The word "machanics" feels a awkward to me, but I can't say why. I feel like "mechanisms" makes more sense.

```suggestion
following mechanisms:
```